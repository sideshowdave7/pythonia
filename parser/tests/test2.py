def a():
 if (not b or c and d):
  e
 else:
  f
def g():
 h; i
 j

