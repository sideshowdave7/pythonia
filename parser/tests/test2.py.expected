(program
 (def
  (a)
  (suite
   (cond ((or (not b) (and c d)) (suite (expr e))) (else (suite (expr f))))))
 (def (g) (suite (begin (expr h) (expr i)) (expr j))))

